src/main/resources/org/jenkins/plugins/lockableresources/actions/LockableResourcesRootAction/tableQueue/table.properties

```diff
@@ -0,0 +1,45 @@
+# The MIT License
+#
+# Copyright 2023 Martin Pokorny.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+queue.isEmpty=There lockable-resources queue is currently empty.
+queue.warning.count=Lockable-resources queue has {0} item(s). The oldest one starts {1} ago!
+queue.warning.count.detail=\
+In many cases the users forgot to unreserve the resource (or external API has die). \
+Sometimes the build does not unlock resource. \
+This might happens when Jenkins die (crashed) and builds are no more runnable after reboot.<br>\
+Please check the status and free the resource manually, when necessary. In case, you has no permission to free the resource, contact your administrator.
```

Comment:
```suggestion
queue.warning.count.detail=\
In many cases, users have forgot to un-reserve the resource (or an external API has died). \
Sometimes the build does not release the resource. \
This can happen when Jenkins crashes and the builds are no longer executable after restart.<br>\
Please check the status and manually release the resource if necessary.
If you do not have permission to release the resource, contact your administrator.
```